custom_components/frigate/icons.py

```diff
@@ -0,0 +1,80 @@
+"""Handles icons for different entity types."""
+
+from xmlrpc.client import Boolean
```

Comment:
Don't need this, you can just use `bool` as the type hint.

Reply:
oops I didn't mean to grab that, thanks!